### ISSUE TITLE
Support Paxmod

### ISSUE THREAD

**Issue by surapunoyousei** [black7375/Firefox-UI-Fix#500]
Paxmod is an addon based on Webextensions unlike TabMixPlus which provides multi-row tabs.

I wanted Lepton to support this addon and have been working on improving compatibility with my own web browser.

First of all, Original and Photon don't work with this addon. It will be as follows.

Use Lepton 
![image](https://user-images.githubusercontent.com/73892113/199517642-fe20fb04-3f38-4a0b-bb9e-c99b72d115de.png)

Use Proton
![image](https://user-images.githubusercontent.com/73892113/199509877-b6d8413d-978c-4ad3-b7e9-b8befdd437c2.png)

As a workaround, consider making the code below optional. This alone should solve the problem. Ultimately, I couldn't find a workaround due to the forcefulness in CSS.

I think you can turn it on by default. It should change quite a bit just by switching and working.

![image](https://user-images.githubusercontent.com/73892113/199515533-df135073-011b-47f7-9ad6-468f69794fec.png)

---
Floorp, the web browser I develop for, bypasses it with JS and CSS. I file this issue because I think it should benefit all users.

https://github.com/Floorp-Projects/Floorp/blob/205fe11909987ed235278eb0561ee40f12ddab11/floorp/browser/base/content/browser-command.js#L413 (The reason why there are two identical ones is that Original and Photon were separated.)
https://github.com/Floorp-Projects/Floorp/tree/ESR102/floorp/browser/themes/designs/lepton

Thank you very much for keeping up the great CSS.

**Comment by black7375** [comment]
It's a amazing project that it's possible only with Webextensions.
I'm going to start an investigation to support [paxmod](https://github.com/numirias/paxmod).

The cases used like [waterfox](https://www.waterfox.net/), [floorp](https://floorp.ablaze.one/) and [running cheese](https://www.runningcheese.com/firefox) are quite important in the Open Source Software, so if possible, I think it is better to supports.

**Comment by surapunoyousei** [comment]
Please let me know if there is anything I can do to help.

Should I submit this issue to Paxmod?

**Comment by black7375** [comment]
I'm waiting for an answer by submitting an issue. https://github.com/numirias/paxmod/issues/100

**Comment by surapunoyousei** [comment]
Finished writing the patch on the Paxmod side.  I think everything will end with the response on the Lepton side.

**Comment by black7375** [comment]
It's very encouraging!!
It seems that webextension can replace AutoConfig(A.K.A userChromeJS [fx-autoconfig](https://github.com/MrOtherGuy/fx-autoconfig) and [firefox-scripts](https://github.com/xiaoxiaoflood/firefox-scripts)) and [Legacy Extensions](https://github.com/xiaoxiaoflood/firefox-scripts/tree/master/extensions) in part.

**Comment by surapunoyousei** [comment]
yes. However, legacy extensions are based on old technology, so Paxmod based on modern technology is fast and relatively secure.

I am happy to have contributed to this project.  Now all that's left to do is wait until the PR is merged.

**Comment by black7375** [comment]
If possible, this project will also be switched to webextension. https://github.com/black7375/Firefox-UI-Fix/discussions/502

**Comment by surapunoyousei** [comment]
Will help. It's not that difficult.

 But I think you'd be better off using a theme.  Firefox still has experimental support for full theming possible with CSS.
 
Extensions are of course also possible.

**Comment by black7375** [comment]
The feature I want is that change the option value in real time, and there is an option related UI.
Perhaps JS is also necessary for sidebar and statusbar, so I was considering using autoconfig, but I just changed the roadmap. #2

**Comment by surapunoyousei** [comment]
Paxmod feels abandoned. I plan to maintain it in the meantime.

**Comment by black7375** [comment]
Yeah, I don't know exactly why, but I also think maintainer's not active right now.

**Comment by surapunoyousei** [comment]
We are currently working on multilingualization of the extension. I will submit a PR when the maintainer returns. Patience until then.

![image](https://user-images.githubusercontent.com/73892113/236394402-b234bf64-7fb6-414a-9645-8485dcb343a2.png)
